css/product-payment.css

```
@@ -0,0 +1,575 @@
+  border: 1px solid var(--color-soft-gray);
+  gap: var(--twenty-pixel);
+  box-shadow: var(--box-shadowL);
+  padding: 12px 12px 12px 10px;
+  gap: var(--ten-pixel);
+  border-radius: var(--eight-pixel);
+  text-decoration: none;
+}
+
+.product-page__warranty{
+  display: flex;
+  align-items: center;
+  font-size: var(--font-size-small);
+  font-weight: var(--font-weight);
+  line-height: var(--eighteen-pixel);
+  letter-spacing: -0.01em;
+  color: var(--color-gray);
+  gap: 25px;
+}
+
+.product-page__requires{
+  font-size: 11px;
+  font-weight: var(--font-weight);
+  line-height: var(--sixteen-pixeles);
+  letter-spacing: -0.01em;
+}
+
+.product-page__penalties{
+  display: flex;
+  gap: var(--twenty-pixel);
+  color: var(--color-striking-gray);
+}
+
+.product-page__input {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.product-page__thanks{
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  border-color: var(--color-whiteX);
+  padding: var(--ten-pixel);
+  padding-right: 30px; 
+  font-size: var(--font-size-smalls);
+  color: var(--color-striking-gray);
+  border-radius: var(--eight-pixel);
+  width: var(--width);
+  cursor: pointer;
+  background: var(--color-whiteX) url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTciIGhlaWdodD0iMTYiIHZpZXdCb3g9IjAgMCAxNyAxNiIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTQuMTA5MzggNkw4LjEwOTM4IDEwTDEyLjEwOTMgNiIgc3Ryb2tlPSIjMzYzNTM1IiBzdHJva2Utd2lkdGg9IjIiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgLz4KPC9zdmc+') no-repeat right 10px center;
+}
+
+.product-page__thanks:focus {
+  border-color: var(--color-pink);
+  outline: none;
+}
+
+.product-page__learn-more{
+  font-size: var(--font-size-small);
+  display: flex;
+  align-items: center;
+  color: var(--color-pink);
+  line-height: var(--sixteen-pixeles); 
+  gap: 7px;
+  margin-top: 2px;
+}
+
+.product-page__bords{
+  border: 1px solid var(--color-solid);
+  gap: 0px;
+  margin: 18px 0px 5px;
+}
+
+.product-page__method{
+  font-size: var(--sixteen-pixeles);
+  font-weight: var(--font-weight);
+  line-height: var(--twenty-pixel);
+  color: var(--color-dark-gray);
+}
+
+.product-page__method-page{
+  display: flex;
+  align-items: center;
+  gap: var(--sixteen-pixeles);
+  margin-top: 10px;
+}
+
+.product-page__ofer{
+  display: flex;
+  align-items: center;
+  gap: var(--ten-pixel);
+  font-size: var(--font-size-smalls);
+  font-weight: var(--font-weight);
+  line-height: var(--eighteen-pixel);
+  letter-spacing: -0.01em;
+  color: var(--color-gray);
+  border: 1px solid var(--color-soft-gray);
+  border-radius: var(--eight-pixel);
+  padding: 14px 15px 14px 10px;
+  text-decoration: none;
+}
+
+.product-page__delivery-type {
+  display: none;
+}
+
+.product-page__delivery-type:checked ~ svg ~ p {
+  color: var(--color-pink);
+  border-radius: var(--eight-pixel);
+}
+
+.product-page__delivery-type:checked ~ svg ~ p::before {
```

Con el `~` el elemento no tiene que estar al lado, simplemente estar despues al mismo nivel
```suggestion
.product-page__delivery-type:checked ~ p::before {
```